openedx/core/djangoapps/user_api/accounts/__init__.py

```diff
@@ -25,3 +28,53 @@
 
 # Indicates the user's preference that all their account information be private.
 PRIVATE_VISIBILITY = 'private'
+
+# Translators: This message is shown when the Unicode usernames are NOT allowed.
+# It is shown to users who attempt to create a new account using invalid characters
+# in the username.
+USERNAME_INVALID_CHARS_ASCII = _(
+    u"Usernames can only contain letters (A-Z, a-z), numerals (0-9), underscores (_), and hyphens (-)."
+)
+
+# Translators: This message is shown only when the Unicode usernames are allowed.
+# It is shown to users who attempt to create a new account using invalid characters
+# in the username.
+USERNAME_INVALID_CHARS_UNICODE = _(
+    u"Usernames can only contain letters, numerals, and @/./+/-/_ characters."
+)
+
+# Translators: This message is shown to users who attempt to create a new account using
+# an invalid email format.
+EMAIL_INVALID_MSG = _(u"Email '{email}' format is not valid.")
```

Comment:
@UmanShahzad CC @brittneyexline @douglashall I tested this PR on my local machine and came across this validation message. When viewed in context (with an actual email address inserted for the placeholder) it sounds a little awkward (to me). For example:

> Email 'a@b@c' format is not valid.

Can we change this to something like:

> "a@b@c" is not a valid email address.

(Also note the double quotes.)